bricks/stm32/main.c

```diff
@@ -95,6 +96,72 @@ static const pbsys_user_program_callbacks_t user_program_callbacks = {
     .stdin_event = user_program_stdin_event_func,
 };
 
+typedef struct _pb_reader_user_data_t {
+    const byte *beg;
```

Comment:
It looks like `beg` is assigned but never ready, so possibly could be removed.

Reply:
Removed, thanks.